lte/gateway/c/core/oai/include/n11_messages_types.h

```diff
@@ -56,16 +61,59 @@ typedef struct qos_flow_level_qos_parameters_s {
   m5g_allocation_and_retention_priority alloc_reten_priority;
 } qos_flow_level_qos_parameters;
 
+typedef enum qos_flow_action_s {
+  policy_action_add = 0,
```

Comment:
By default integer constants within enum are initialized with 0 and sequentially incremented.  Not required to assign the values here